_posts/2020-12-01-infrastructure-migration-finished.md

```diff
@@ -0,0 +1,275 @@
+---
+title: Infrastructure migration finished
+author: TrueBrain
+---
+
+Today is a good day.
+Today is the day all services of openttd.org run on [AWS](https://aws.amazon.com/).
+Today is the day I can finally shut down our [OVH dedicated server](https://www.ovhcloud.com/en-gb/bare-metal/).
+Today is the day I can finally retire software like:
+- [Django](https://www.djangoproject.com/) 1.2 (released May 2010)
+- [Debian](https://www.debian.org/) Etch (release April 2007)
+- An IRC bot that was running since 2016.
+
+This was the longest migration I ever did: 11 months in the making.
+But we finally did it!
+
+This was mostly made possible by us receiving [AWS Open Source credits](https://aws.amazon.com/blogs/opensource/aws-promotional-credits-open-source-projects/).
+This allowed me to try out AWS services, do stupid things (rack up a bill of more than 500 dollar per month .. which is HUGE for us), and all while knowing it wouldn't clean out our donation-funds (read: the credits covered all our costs for 2020).
+Sadly, in 2021 we will not receive AWS Open Source credits .. this means no more experimenting.
+So a perfect time to write up what happened over the last 16 years, and more specifically, last 2 years.
+
+<!-- more -->
+
+First of all, this will be a long post.
+Just .. so you are prepared.
+
+We start with a bit of history:
+
+In 2004 openttd.org was a website, and a wiki, and that was about it.
+In the years after, we greatly extended what we offered on openttd.org:
+- Master server, so you have in-game server listing
+- Content services, so you can download NewGRFs, Scenarios, ... in-game
```

Comment:
The "..." here reads a little like a pause for "in-game", rather than a continuation of the list (to me)
```suggestion
- Content services, so you can download NewGRFs, Scenarios, and so on from in-game
```